topic_tools/package.xml

```diff
@@ -26,6 +26,10 @@
 
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>std_msgs</test_depend>
+  <test_depend condition="$ROS_DISTRO == humble" version_gt="0.9.2">rosidl_runtime_py</test_depend>
+  <test_depend condition="$ROS_DISTRO == rolling" version_gte="0.11.0">rosidl_runtime_py</test_depend>
```

Comment:
@mllofriu are these really needed? For example, if you build against Humble, I would expect that you'd get an older version of `rosidl_runtime_py`.

Reply:
I'll take a look. I'm actually using humble (docker). Do the tests run against rolling in github?

Reply:
Yeah they run against Rolling, but like I said I don't think you need to specify the version here.

Reply:
Done